plugin/src/main/java/com/microsoft/java/bs/gradle/plugin/SourceSetsModelBuilder.java

```diff
@@ -108,10 +112,13 @@ public Object buildAll(String modelName, Project rootProject) {
         List<File> compileClasspath = new LinkedList<>(sourceSet.getCompileClasspath().getFiles());
```

Comment:
I'm thinking that if we need to try-catch here. Though the method signature does not declare that it will throw exception.

Will `sourceSet.getCompileClasspath().getFiles()` throw RuntimeException? (maybe a question to @donat @reinsch82 @hegyibalint)

Reply:
I don't see anything that would through there.

Reply:
Yes, it was the case then. [FileCollection#getFiles](https://docs.gradle.org/current/javadoc/org/gradle/api/file/FileCollection.html#getFiles()) used to throw an exception. I will wrap it in try-catch.

Reply:
I'll handle it in https://github.com/microsoft/build-server-for-gradle/pull/163